libs/category/driver/magento/src/category.service.ts

```diff
@@ -91,6 +89,38 @@ export class DaffMagentoCategoryService implements DaffCategoryServiceInterface
     );
   }
 
+  getByUri(categoryRequest: DaffCategoryUriRequest): Observable<DaffGetCategoryResponse> {
+    const truncatedUri = categoryRequest.uri.replace('.html', '');
```

Comment:
I don't think we should be truncating the URI.

Reply:
I have a PR out for appending the suffix to the URL, which I think it required? Or do you think that we should have the URL suffix in the daff urls at all?

Reply:
Why would magento expect a url without the suffix, but provide the suffix in the category response? Would it be better for daffodil to only deal with whatever format the platform expects in the uri query? It seems like with this PR and the suffix PR the end result is just us putting the suffix onto the uri only to remove it just before the request is made. Do we need the suffix for some reason?

Reply:
yeah I was under the impression that there was a compelling reason to need the suffix app-side. There is some consequence for routing behavior but I don't remember what it is. @damienwebdev do you know?

Reply:
In retrospect, it's more correct to say that this truncating is configurable by the app-dev. It's not a safe assumption that the suffix is `.html`.

Reply:
changed truncation behavior to be dynamic based on config option